Makefile

```diff
@@ -152,3 +153,23 @@ gen-crd: init-codegen ## Generate WebhookConfiguration, ClusterRole and CustomRe
 .PHONY: generate
 generate: init-codegen ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
 	hack/update-codegen.sh
+
+.PHONY: install-crd
+install-crd: gen-crd ## Install CRDs into the K8s cluster specified in ~/.kube/config.
```

Comment:
I do not think we need this; we use helm to install all stuffs

Reply:
ok，it has been removed